.github/workflows/check-pr.yml

```diff
@@ -51,6 +51,81 @@ jobs:
         with:
           image: ghcr.io/podkrepi-bg/api/migrations:pr
 
+  build-migrations-docker-image:
+    name: Build migrations docker image
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Login to GitHub Packages Docker Registry
```

Comment:
this shouldn't be here. We are not pushing images from PRs since this is potentially unreviewed code

Reply:
this was only temporary from the release.yml for testing the build, removed now. If anything needs to be fixed it needs to be updated in release.yml